lib/engine/db/sqlite/schema.sql

```diff
@@ -31,3 +31,15 @@ create table channel (
        uniqueId varchar(255) primary key,
        value text default null
 );
+
+drop table if exists conversation ;
+
+create table conversation (
+       uniqueId varchar(255) primary key,
+       context text default null,
+       agent text default null,
+       agent_target text default null,
+       intermediate_context text default null,
+       user text default null,
+       user_target text default null
+);
```

Comment:
You'll want to add the migration code in db/sqlite/migrations. Add a function `migrateTo9` to the MIGRATION array that will create the table.